pkg/client/didexchange/client.go

```diff
@@ -67,6 +68,9 @@ type protocolService interface {
 	// CreateImplicitInvitation creates implicit invitation. Inviter DID is required, invitee DID is optional.
 	// If invitee DID is not provided new peer DID will be created for implicit invitation exchange request.
 	CreateImplicitInvitation(inviterLabel, inviterDID, inviteeLabel, inviteeDID string) (string, error)
+
+	// SaveInvitation saves given invitation in did-exchange service connection store
+	SaveInvitation(invitation *didexchange.Invitation) error
```

Comment:
Why is this needed in client?

Perhaps it move again with the next task.

Reply:
'CreateInvitation' logic is actually inside didexchange client.
So DIDExchangeService exports SaveInvitation() function to retain `save` logic inside service.
Again, this logic will be removed once writable '*connectionStore' is moved out of didexchange.